src/phoenix/server/api/routers/v1/spans.py

```diff
@@ -116,6 +121,13 @@ async def query_spans_handler(
     if not results:
         raise HTTPException(status_code=HTTP_404_NOT_FOUND)
 
+    if accept == "application/json":
```

Comment:
There are some conventions around accept headers we might want to follow, e.g., if multiple media types are provided via a comma-delimited string. Non-blocking. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept